### PR TITLE
Keyboard shortcut

### LINKED ISSUE #34: Keyboard shortcut to run code
https://github.com/securingsincity/react-ace/blob/master/docs/FAQ.md#how-do-i-add-key-bindings

### PR DESCRIPTION
**Pull request by mjenrungrot**
Use `Command-Enter` for Mac and `Ctrl-Enter` for Windows. (To resolve #34)

### CODE REVIEW

File: src/components/Editor.js [hunk 1]
```diff
@@ -75,6 +75,15 @@ class Editor extends Component<Props, State> {
         onLoad={this.onEditorLoad}
         {...aceStyleProps}
         markers={this.getMarkers()}
+        commands={[
+          {
+            name: 'run',
+            bindKey: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
+            exec: () => {
```
Comment (jcruz) [inline src/components/Editor.js]: this can be reduced to `exec: this.handleClick,`

**Review by jcruz — COMMENTED** [review]
Nice! I'd make a `getCommands` function to reduce the clutter in the render function. Only issue is that this does not prevent users from making multiple requests when loading so just add a check in `handleClick`.

File: src/components/Editor.js (hunk 1 above)
Comment (mjenrungrot) [reply src/components/Editor.js]: Got this on the browser's console.
```
Warning: Cannot update during an existing state transition (such as within `render` or another component's constructor). Render methods should be a pure function of props and state; constructor side-effects are an anti-pattern, but can be moved to `componentWillMount`
```
Edit: Fixed it.

File: src/components/Editor.js [hunk 2]
```diff
@@ -65,6 +65,14 @@ class Editor extends Component<Props, State> {
   };
 
   render() {
+    const editorBindedComamnds = [
```
Comment (jcruz) [inline src/components/Editor.js]: Still want this outside of `render`

Comment (mjenrungrot) [reply src/components/Editor.js]: Need to call `this.handleClick`

Comment (jcruz) [reply src/components/Editor.js]: a function within the class should still have access

**Comment by mjenrungrot** [comment]
@jcruz Done.

**Review by jcruz — COMMENTED** [review]
In `handleClick` make sure it's not loading before doing the run request

**Review by jcruz — APPROVED** [review]

**Pull request merged**